.npmignore

```diff
@@ -3,6 +3,7 @@ coverage
 
 .npmignore
 .gitignore
+.nyc_output
```

Comment:
using `"files"` in `package.json` is nicer than having to maintain this

Reply:
Yeah good call, will switch it over.